shardingsphere-infra/shardingsphere-infra-context/src/main/java/org/apache/shardingsphere/infra/context/metadata/MetaDataContextsBuilder.java

```diff
@@ -139,14 +135,4 @@ private ShardingSphereResource buildResource(final DatabaseType databaseType, fi
     private ShardingSphereSchema buildSchema(final DatabaseType databaseType, final Map<String, DataSource> dataSourceMap, final Collection<ShardingSphereRule> rules) throws SQLException {
         return SchemaBuilder.build(new SchemaBuilderMaterials(databaseType, dataSourceMap, rules, props));
     }
-
-    private Collection<ShardingSphereUser> getUsersFromAuthorityRule() {
```

Comment:
TODO-2,
Please consider whether `shardingsphere-infra-authority-api` can be removed from `shardingsphere-infra/shardingsphere-infra-context/pom.xml`